src/content/popup/KanjiReferencesTable.fixture.tsx

```diff
@@ -0,0 +1,176 @@
+import { KanjiReferencesTable } from './KanjiReferencesTable';
+
+// import '../../../css/popup.css';
```

Comment:
So this is super tricky, but it looks like we're using `options.html` as the template file due to this line here:

https://github.com/birchill/10ten-ja-reader/blob/9a0ce777f8a17070094523c329f2162ff92d9dd2/webpack.config.cosmos.js#L51

That's why we don't need to import `popup.css` ourselves.

Ideally we'd use a separate template for popup components vs options components but I don't know if there's a way to get Cosmos to use separate template files based on the fixture. It might be worth having a quick dig (or asking on their discord) but otherwise I guess this is fine as-is.

Reply:
I can definitely see if I can set up a separate template for options and popup fixtures.

FYI, the commented out line is there because the coloration on the table rows wasn't working, and I thought it was because the `popup.css` styles weren't getting imported properly. Turns out that it was actually because `theme.css` hadn't been imported, since that's where the CSS var for `--cell-highlight-bg` gets defined. Just hadn't gotten to removing it yet :).

Reply:
FYI, looks like it's [not possible](https://discord.com/channels/620737684859781150/1235330430882938931/1235338431895175309) to have more than one HTML template. The main React Cosmos maintainer Ovidiu recommended we import the CSS via JS, and use decorators if we need to vary it based on folder.

Reply:
Thanks for chasing up with Ovidiu about that! Ok, that seems fine. It's not strictly the CSS files that I'm interested in but rather the SVG assets in `options.html` (which should be available to options components but not to popup componets). Anyway, using `options.html` as-is seems fine for now. If it becomes a problem we can address it then.